ratis-server/src/main/java/org/apache/ratis/server/impl/ServerImplUtils.java

```diff
@@ -47,26 +47,28 @@ private ServerImplUtils() {
   /** Create a {@link RaftServerProxy}. */
   public static RaftServerProxy newRaftServer(
       RaftPeerId id, RaftGroup group, RaftStorage.StartupOption option, StateMachine.Registry stateMachineRegistry,
-      RaftProperties properties, Parameters parameters) throws IOException {
+      RaftProperties properties, Parameters parameters, ThreadGroup threadGroup)
```

Comment:
Please reorder the parameters:
```java
ThreadGroup threadGroup, RaftProperties properties, Parameters parameters)
```

Reply:
Rearranged together with all the places that have the same argument list